framework/db/BaseActiveRecord.php

```diff
@@ -311,8 +311,7 @@ public function __get($name)
     public function __set($name, $value)
     {
         if ($this->hasAttribute($name)) {
-            if (
-                !empty($this->_relationsDependencies[$name])
+            if (!empty($this->_relationsDependencies[$name])
```

Comment:
Got worse.

Reply:
I first applied PSR2. Now I force pushed with PSR12 and this line is not changed anymore.